1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,47 @@
 function highlightWords(paragraph, colours) {
   // Write your code here...
+  const contentDiv = document.getElementById("content");
+
+  const pElement = document.createElement("p");
+  const selectElement = document.createElement("select");
+
+  //default option at first
+  const defaultOption = document.createElement("option");
```

Comment:
Functinally,  this is fine but the function itself is getting quite big and has multiple responsibilities (render select, render words). Can you try breaking it down so that functions have distinct responsibilities?

If you have time, you can look into single responsibility principle which is the first one of SOLID principles (do not worry about the rest for now)